tfhe/src/integer/server_key/radix_parallel/tests_unsigned.rs

```diff
@@ -2962,3 +2963,41 @@ where
         );
     }
 }
+
+fn integer_trim_msb_blocks_handles_dirty_inputs<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let param = param.into();
+    let (client_key, server_key) = crate::integer::gen_keys_radix(param, NB_CTXT);
+    let modulus = (param.message_modulus().0 as u64)
+        .checked_pow(NB_CTXT as u32)
```

Comment:
let's add a comment near NB_CTXT that it needs to relatively small or never change otherwise the shifts line 2978 and 2979 will be wrong in the future

or hard code some small values guaranteed to not overflow for this test

Reply:
yep sounds good